src/scene/visitors/elementrgbbitmapsetter.cpp

```diff
@@ -26,7 +31,7 @@ void ElementRGBBitmapSetter::visit(AdafruitGFXElement* el) {
 
         for (uint16_t x = 0; x < el->width(); x++) {
             for (uint16_t y = 0; y < el->height(); y++) {
-                el->drawPixel(x, y, cur_bitmap->getPixel(x, y));
+                el->drawPixel(x, y, crgbToRGB565(cur_bitmap->getPixel(x, y)));
```

Comment:
Good catch here! I hadn't used the adafruit elements myself in a while so I didn't know they were broken! (I need to set up some tests at some point I guess)